lib/aux_js.cpp

```diff
@@ -54,6 +54,9 @@ bool startVisualization(const std::string input, const std::string data_type,
    std::stringstream ss(input);
    const int field_type = ReadStream(ss, data_type);
 
+   // reset antialiasing
+   GetAppWindow()->getRenderer().setAntialiasing(0);
```

Comment:
@kanye-quest, is there a better way to do this? Without it, the antialiasing status was not updating...